init.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import "github.com/kklipsch/cli"
+
+func Initialize(c *cli.Context) error {
```

Comment:
Its worth noting that Go actually has an "init" entry point.  Each package may define an init function to be called exactly once if its imported.  That is the reason you see imports that start with an underscore (import _ "somepackage").  That is, to paraphrase, "importing for the side effects only".

I say all this to make for two reasons:
1. to make you cringe.
2. so you are aware and don't define something that doesn't work like you expect.  This function is fine.  It is specifically "func init() {}" that you have to watch out for.

http://golang.org/ref/spec#Package_initialization